imap_processing/codice/generate_ccsds_header_xml.py

```diff
@@ -3,48 +3,48 @@
     the CCSDS header parameters in the XTCE file.
     """
 
-import xml.etree.ElementTree as ET
+import xml.etree.ElementTree as Et
```

Comment:
For things like this, I might actually suggest sticking with `ET` and then ignoring this error in the ruff config. The main python docs use ET, so sticking with convention is sometimes better than appeasing a linter.
https://docs.python.org/3/library/xml.etree.elementtree.html

I'm fine either way, but just saying that I totally understand why you did it this way previously and also why the linter is complaining :) Sometimes there is no winning 😂 

https://beta.ruff.rs/docs/settings/#per-file-ignores
It looks like it was `N817` for this case.